src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs

```diff
@@ -61,6 +61,7 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
             MNM.ExpressRouteCircuitSkuTier.Standard,
             MNM.ExpressRouteCircuitSkuTier.Premium,
             MNM.ExpressRouteCircuitSkuTier.Basic,
```

Comment:
Do you need to communicate to customers that `Basic` is being deprecated?

Reply:
Yes, the PMs will communicate this broadly. We have 2 customers using Basic, hence keeping it here. We will work with the 2 customers to move to Local.